java/java/src/main/java/org/signal/internal/Native.java

```diff
@@ -99,6 +102,8 @@ private Native() {}
   public static native long[] IdentityKeyPair_Deserialize(byte[] data);
   public static native byte[] IdentityKeyPair_Serialize(long publicKey, long privateKey);
 
+  public static native void Logger_Initialize(int maxLevel, Class loggerClass);
```

Comment:
I wasn't quite sure what naming convention to go with here, but this didn't seem too bad to me?